pkg/project/api/validation/validation.go

```diff
@@ -12,12 +11,14 @@ import (
 )
 
 // ValidateProject tests required fields for a Project.
-func ValidateProject(project *api.Project) fielderrors.ValidationErrorList {
+func ValidateProject(project *api.Project, create bool) fielderrors.ValidationErrorList {
```

Comment:
I don't think you need to change this signature.

On create we call:
https://github.com/openshift/origin/blob/master/pkg/project/registry/project/rest.go#L57

On update we call:
https://github.com/openshift/origin/blob/master/pkg/project/registry/project/rest.go#L67